src/Events/Integrations/Plugins/Event_Tickets/Provider.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Event_Tickets;
+
+use TEC\Events\Integrations\Integration_Abstract;
+use TEC\Events\Integrations\Plugins\Plugin_Integration;
+
+/**
+ * Class Provider
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Event_Tickets
+ */
+class Provider extends Integration_Abstract {
+	use Plugin_Integration;
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_slug(): string {
+		return 'event-tickets';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function load_conditionals(): bool {
+		return function_exists( 'tribe_tickets' );
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function load(): void {
+		$this->load_tickets_emails_integration();
+	}
+
+	/**
+	 * Loads the Tickets Emails integration.
+	 *
+	 * @since TBD
+	 */
+	public function load_tickets_emails_integration() {
```

Comment:
Unless there is a reason not to I'm not seeing?

Reply:
Because we're handling integrations with themes and plugins, not features per themes and plugins. If a tool is overused and not used for what's meant to, then it becomes irrelevant and confusing.

If I extend that, then I have to invent a slug and a type (so far, from what I saw, the integrations were meant for themes and plugins, and not features per theme or plugin).

Reply:
I've jumped in here to make these modifications. I was pending doing some changes to move the integrations suite to common.